test/TestUtilities/Test.Utility/Signing/CertificateStoreUtilities.cs

```diff
@@ -18,5 +18,29 @@ public static StoreLocation GetTrustedCertificateStoreLocation(bool readOnly = f
             }
             return (RuntimeEnvironmentHelper.IsWindows || RuntimeEnvironmentHelper.IsMacOSX) ? StoreLocation.LocalMachine : StoreLocation.CurrentUser;
         }
+
+        internal static StoreName GetCertificateAuthorityStoreName()
+        {
+            // According to https://github.com/dotnet/runtime/issues/48207#issuecomment-778293907,
+            // only My, Root (RO), and Disallowed stores work on macOS.
+            // So use different approaches for Windows, Mac and Linux.
+
+            return (RuntimeEnvironmentHelper.IsWindows || RuntimeEnvironmentHelper.IsLinux) ? StoreName.CertificateAuthority : StoreName.My;
+        }
+
+        internal static StoreName GetTrustedCertificateStoreNameForLeafOrSelfIssued()
+        {
+            // On MacOs, if we add the leaf or self-issued certificate into LocalMachine\Root, the private key will not be accessed.
+            // So the dotnet signing command tests will fail for:
+            //  "Object contains only the public half of a key pair. A private key must also be provided."
+            return StoreName.My;
+        }
+
+        internal static StoreLocation GetTrustedCertificateStoreLocatioinForLeafOrSelfIssued()
```

Comment:
`Locatioin` -> `Location`

Append `Certificate` to method name.

Reply:
Thanks! Fixed.